client/js/components/ParkList.jsx

```diff
@@ -13,7 +13,7 @@ export default class ParkList extends React.Component {
     render() {
         var parkList = this.props.parks.map((park) => {
             return (
-                <div className='row' onClick={() => this.selectPark(park)} key={park.park_id}>
+                <div className='row u-clickable' onClick={() => this.selectPark(park)} key={park.park_id}>
```

Comment:
Do you think it would be more semantic to make these `<button>`s or `<a>`s instead?